nim-base/Dockerfile

```diff
@@ -1,13 +1,10 @@
 FROM statusteam/debian-pre-nim
 MAINTAINER Yuriy Glukhov <iurii@status.im>
 ENV PATH $PATH:/nim/bin:/root/.nimble/bin
-RUN git clone https://github.com/nim-lang/nim.git \
+RUN git clone https://github.com/status-im/nim.git \
     && cd nim \
-    && git remote add statusim https://github.com/status-im/nim.git \
```

Comment:
`git checkout status` is missing

Reply:
shouldn't be necessary - it's the default branch for that repo so will get checked out automagically..

Reply:
ah i see.